src/lib/layouts/workflow-run-layout.svelte

```diff
@@ -133,32 +134,42 @@
   });
 </script>
 
-<LabsModeGuard>
-  <div
-    class="absolute bottom-0 left-0 right-0 {$viewDataEncoderSettings
-      ? 'top-[540px]'
-      : 'top-0'}
+{#if raw}
+  <div class="h-auto whitespace-break-spaces rounded bg-primary p-4">
+    {JSON.stringify(
```

Comment:
Would it be useful to add a copy button here so users can copy the JSON and inspect it elsewhere?

Reply:
Yeah that's a good idea. These can be pretty huge (~50mb) but there's no limit to copyboard copy size.